docs/pages/enroll-resources/server-access/guides/host-user-creation.mdx

```diff
@@ -218,6 +218,39 @@ If multiple entries are specified in the `host_user_uid` or `host_user_gid` only
 
 </Admonition>
 
+For Teleport users that don't have a `host_user_uid` trait, starting from Teleport 17.3, it's possible to configure the cluster to assign the same UID for any given username for automatically created host users across all Teleport SSH instances.
```

Comment:
Should we mention version numbers from previous major versions? If we do, should we get rid of the version numbers once we're past the version compatibility window? In this case we could safely get rid of the numbers in the v19 docs, for example.